VisualizationBase/src/overlays/ArrowOverlay.cpp

```diff
@@ -79,4 +96,22 @@ void ArrowOverlay::updateGeometry(int, int)
 					 second->scenePos().y() + second->heightInScene() / 2 - leftTopCorner.y()}.toPoint();
 }
 
+void ArrowOverlay::hoverEnterEvent(QGraphicsSceneHoverEvent *) {
```

Comment:
Yes, always :) and if you look at the base class of `GenericHandler` (I believe `InteractionHandler`) you will find the appropriate virtual methods regarding hovering that you'll need to override.